WcaOnRails/app/javascript/edit-events/modals/Cutoff.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+
+import events from 'wca/events.js.erb'
+import formats from 'wca/formats.js.erb'
+import AttemptResultInput from './AttemptResultInput'
+import {
+  pluralize,
+  matchResult,
+  roundIdToString,
+  attemptResultToString,
+} from './utils'
+
+function parseRoundId(roundId) {
+  let [eventId, roundNumber] = roundId.split("-");
+  return { eventId, roundNumber };
+}
```

Comment:
There are three other places in this diff using `split("-")`, let's make use of the new function! =)